packages/features/ee/round-robin/assignmentReason/AssignmentReasonRecorder.ts

```diff
@@ -13,11 +13,15 @@ export default class AssignmentReasonRecorder {
     routingFormResponseId,
     organizerId,
     teamId,
+    isRerouting,
+    reroutedByEmail,
```

Comment:
We can reuse the logic that we use when recording the initial routing. That way we can see what attributes changed.